main.py

```diff
@@ -0,0 +1,52 @@
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+import models, schemas, crud
+from database import SessionLocal, engine
+
+models.Base.metadata.create_all(bind=engine)
+
+app = FastAPI()
+
+
+def get_db():
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@app.post("/authors/", response_model=schemas.Author)
```

Comment:
```suggestion
@app.post("/author/", response_model=schemas.Author)
```
Ми створюємо одного автора. Хоча ця зміна не є обовʼязковою.